.flowconfig

```diff
@@ -1,58 +0,0 @@
-[ignore]
```

Comment:
That's to use [flow](https://flow.org/en/docs/config/) with JS, but we don't need because we already use a typed language :)